pkg/sfu/receiver.go

```diff
@@ -613,3 +611,10 @@ func (w *WebRTCReceiver) GetPrimaryReceiverForRed() TrackReceiver {
 	}
 	return w.primaryReceiver.Load().(*RedPrimaryReceiver)
 }
+
+func (w *WebRTCReceiver) GetTemporalLayerFpsForSpatial(layer int32) []float32 {
+	if !w.isSVC {
```

Comment:
Should this be `if w.isSVC`?

Reply:
In non svc encoding, every spatial layer have a standalone buffer, in its own buffer, the spatial is treat as 0 by frame calculate (in ExtPacket, spatial is invalid layer ).

Reply:
Got it.